configure.ac

```diff
@@ -94,9 +89,19 @@ AC_FUNC_MALLOC
 AC_FUNC_MMAP
 AC_CHECK_FUNCS([gettimeofday memmove memset munmap strstr strtoul strtoull])
 
-AC_CONFIG_FILES([Makefile
-                 test/Makefile
-		 test/hisi_hpre_test/Makefile
-		 test/hisi_sec_test/Makefile
-                 test/hisi_zip_test/Makefile])
+AC_CONFIG_FILES([Makefile])
+AS_IF([test -n "$with_uadk_v1"],
+	AC_CONFIG_FILES([v1/Makefile]),
+	AC_CONFIG_FILES([test/Makefile
+			 test/hisi_hpre_test/Makefile
+			 test/hisi_sec_test/Makefile
+			 test/hisi_zip_test/Makefile])
+)
+
+#AC_CONFIG_FILES([Makefile
+#         test/Makefile
+#	 test/hisi_hpre_test/Makefile
+#	 test/hisi_sec_test/Makefile
+#         test/hisi_zip_test/Makefile
+#	 v1/Makefile])
```

Comment:
can we remove above comments?

Reply:
No, we can't remove them. Without them, Makefile can't be generated successfully.